airflow/configuration.py

```diff
@@ -1548,10 +1548,7 @@ def initialize_config() -> AirflowConfigParser:
         if local_conf.getboolean("core", "unit_test_mode"):
             local_conf.load_test_config()
 
-    # Make it no longer a proxy variable, just set it to an actual string
-    global WEBSERVER_CONFIG
-    WEBSERVER_CONFIG = AIRFLOW_HOME + "/webserver_config.py"
-
+    WEBSERVER_CONFIG = local_conf.get("webserver", "config_file")
```

Comment:
Needs formatting, but I think we need: 

```suggestion
    WEBSERVER_CONFIG = local_conf.get("webserver", "config_file", 
                   fallback= AIRFLOW_HOME + "/webserver_config.py")
```

Otherwise it will fail for those who have the old config file

Reply:
(or is it aautomaticaly handled with the default?

Reply:
It's automatically handled. Most fallbacks can go honestly, much more useful in providers than core.

Reply:
Yeah I vaguely recall that - I wanted to clean up the code doing it once and failed miserably :)

Reply:
Thanks for clarifying this. Was not so aware about this